source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -5,21 +5,245 @@ Enabling Translations on a New Repo
 .. contents::
  :local:
  :depth: 2
- 
-===========
-Python Repo
-===========
 
-TODO: How to enable translations infrastructure on a Python repo
+==========
```

Comment:
Please use the heading structure describe in the Documentation quick-reference instead of using double headings.  https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#headings